scripts/irods/test/test_misc.py

```diff
@@ -256,3 +259,31 @@ def test_iinit_does_not_crash_from_long_home_directory__issue_5411(self):
 
         # Remove temp directory
         shutil.rmtree(temp_dir_name)
+
+    @unittest.skipIf(plugin_name == 'irods_rule_engine_plugin-python', 'native rule used')
+    def test_msiFlushMonStat_clears_database_rows__issue_6350(self):
+
+        def run_rule_code(rule_code):
+            self.admin.assert_icommand(['irule','-r','irods_rule_engine_plugin-irods_rule_language-instance', rule_code, 'null', 'null'])
+
+        def get_number_of_serverload_rows():
+            database_select_command = ['psql', 'ICAT', '-t', '-c', 'select count(create_ts) from r_server_load']
+            (rc, out, _) = outp = assert_command(database_select_command, 'STDOUT')
+            return int(out,10)
+
+        # Create a row in the database and mark a time for the ageoff window.
+        run_rule_code("msiServerMonPerf('default','1')")
+        keep_time = int(time.time())+3
```

Comment:
I didn't catch that - thanks.  Actually the effect will probably be the same ( int(x)+N usually == int(x+N) since int just rounds down), but it will be less confusing to make the two lines more consistent.

Reply:
As a matter of curiosity, it appears the two expressions are unequal in Python if say N==1 and x is in (-1,0), but this will never happen with time.time( )

Reply:
I'm sure you're correct. Let's make them use the same math so no one is confused later though.

Reply:
done